### ISSUE TITLE
Add support for non-FITS tiled imagery in Research App

### ISSUE THREAD

**Issue by astrodavid10** [WorldWideTelescope/wwt-webgl-engine#186]
Currently non-FITS tiled imagery will not successfully load in the research app.

**Comment by pkgw** [comment]
Now that I think of this ... you primarily want this because you want to share tiled FITS datasets with the general public and include other non-FITS for context/comparison, right? It does feel like we've been getting some great mileage from sharing FITS datasets that haven't necessarily been converted into RGB PR images (yet).

What I'm thinking is that for this particular use case, maybe it would be better to think about better FITS support in the embed app, rather than better non-FITS support in the research app. In particular, the research app UX is intended to be aimed at (more) expert users, so it may use more jargon and have more complex controls than the embed app UX.

That isn't to argue that the research app *shouldn't* have better support for non-FITS images, but if the goal is functionality aimed at the general public, the design vision is that it should go in the embed app. For instance, the best UX to let a non-specialist "play with" a FITS file is probably going to look at least a little different than one aimed at researchers doing pixel-level analysis.

**Comment by pkgw** [comment]
CC @Carifio24 for visibility, please chime in if you have any thoughts!

**Comment by astrodavid10** [comment]
@pkgw Yes to your first question and yes I think it would be great to have more simplistic FITS control in the embed app. 

My first priority would be the ability to layer images in the embed app as can be done in the research app. As in, I'd like to be able to compare more than just a background and foreground image - I'd ideally like to be able to layer several for multiwavelength / different telescope / temporal difference comparison.
